ts/common/context/DSExperimentalContext/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { useIOExperimentalDesign } from "@pagopa/io-app-design-system";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { useLayoutEffect } from "react";
+
+/**
+ * This hook is used to enable the experimental design system reading the value from AsyncStorage
+ * remove this once the experimental design system is stable
+ */
+export const useStoredExperimentalDesign = () => {
+  const { setExperimental } = useIOExperimentalDesign();
+
+  useLayoutEffect(() => {
+    AsyncStorage.getItem("isDesignSystemEnabled")
```

Comment:
Is not it better if we group together the functions that get/set the value from/on the Async Storage? Maybe in an util file where they can also share the `isDesignSystemEnabled` key

Reply:
It could be useful, but consider that those must be the only 2 points where the value should be handled, I would prefer to keep it boxed and not exported for further usages. This hook and value should be ignored in future, once the new DS appearance will be the chosen one

Reply:
In such case, I would suggest to just export and share the `isDesignSystemEnabled` key

Reply:
[b099489](https://github.com/pagopa/io-app/pull/5064/commits/b099489fc2684ca5f1bec892070f20e757185b86) to solve